spec/course_spec.rb

```diff
@@ -0,0 +1,31 @@
+require './lib/course'
+require './lib/student'
+
+RSpec.describe Course do
+  describe '#initialize' do
+    it 'exists' do
+      course = Course.new("Calculus", 2) 
+
+      expect(course).to be_an_instance_of(Course)
+      expect(course.name).to eq("Calculus")
+      expect(course.capacity).to eq(2)
+      expect(course.students).to eq([])
+      expect(course.full?).to be false
+    end
+  end
+
+  describe '#enrollement' do
+    it 'checks enrollment' do
+      course = Course.new("Calculus", 2) 
+      student1 = Student.new({name: "Morgan", age: 21})
+      student2 = Student.new({name: "Jordan", age: 29})
+
+      course.enroll(student1) 
+      course.enroll(student2) 
+
+      expect(course.students).to eq([student1, student2])
+      expect(course.full?).to be true
+
+    end
```

Comment:
this test returns a false positive; be sure to also check for a false value returned from the `full?` method.
```suggestion
    it 'checks enrollment' do
      course = Course.new("Calculus", 2) 
      student1 = Student.new({name: "Morgan", age: 21})
      student2 = Student.new({name: "Jordan", age: 29})

      course.enroll(student1) 
      expect(course.full?).to be false
      course.enroll(student2) 

      expect(course.students).to eq([student1, student2])
      expect(course.full?).to be true

    end
```

Reply:
That makes sense. I will pay attention to testing edge cases (if that is what this was called)